activate-org.sh

```diff
@@ -1,24 +1,52 @@
 #!/bin/sh
 
+# Activates an organization by setting its suspended status to false and starts
+# all stopped applications in all spaces of the organization.
+
+# Requires cf CLI version 7+ to work.
```

Comment:
If the org developers are using cf cli v6, will us launching the apps in v7 affect their ability to manage the apps?

Reply:
No, it shouldn't; the requirement here is because of the switch to the V3 APIs and I wanted to make sure we're using compatible tools on our side when running things.

Reply:
👍